airbyte-config/models/src/main/resources/types/StandardWorkspace.yaml

```diff
@@ -9,6 +9,7 @@ required:
   - name
   - slug
   - initialSetupComplete
+  - tombstone
```

Comment:
in the [code](https://github.com/airbytehq/airbyte/pull/2325/files#diff-f6c08873b6a8e35518a1dcc316cbc95975632c8ff27908ecffc425472b95b80bR77) i think you already check that if tombstone is not set that we treat it as false. i think that means that this field then doesn't need to be required. making it not required _should_ mean that we should be able to not need to do a migration. 

Up to you, if you feel like it should be required, then we can keep the migration and just drop the check you're doing in the config repository. LMK if that's not clear.